pyrandwalk/pyrandwalk_obj.py

```diff
@@ -84,20 +84,30 @@ def prob_sec(self, sequence, initial_dist=None):
             current_state = next_state
         return probability
 
-    def run(self, ntimes=10, show=False):
+    def run(self, ntimes:int=10, show:bool=False, starting_state:int=None):
         """
         Run random walk for ntimes and print out result on each transition.
 
         :param ntimes: numbers of running
         :type ntimes: int
         :param show: flag which is set when showing run is desired
+        :starting_state: the state from which the randon walk should start from
```

Comment:
change it to:
```
:param starting_state: the state from which the random walk should start from
:type starting_state: None
```

Reply:
Sorry i missed 'param' before the arg 😅. Will fix this : )